bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -25,8 +25,12 @@ spec:
         {{- if .Values.postgresql.podLabels }}
         {{- include "postgresql-ha.tplValue" (dict "value" .Values.postgresql.podLabels "context" $) | nindent 8 }}
         {{- end }}
+      annotations:
```

Comment:
This part is going to be added to the deployment even if the podAnnotations and metrics.annotations are empty, can you include this part inside a condition? Something like
```
{{- if or .Values.postgresql.podAnnotations .Values.metrics.annotations }}
annotations:
...
```

Reply:
I test this feature with Kubernetes v1.10.0 and Helm v3.0.1 and if we don't have any `postgresql.podAnnotations` and `metrics.annotations` we don't have any annotations in our Kubernetes deployments.
Do you want something like this or something else?

Reply:
For example, running `helm template`, in the statefulset we can see an empty `annotations` block:
```yaml
# Source: postgresql-ha/templates/postgresql/statefulset.yaml
apiVersion: apps/v1
kind: StatefulSet
...
  template:
    metadata:
      labels:
        app.kubernetes.io/name: postgresql-ha
        helm.sh/chart: postgresql-ha-3.5.4
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/component: postgresql
      annotations:
    spec:
      securityContext:
        fsGroup: 1001
...
```
with the suggested approach, this section shouldn't appear when the annotations are not set

Reply:
Done.